test/LD.c

```diff
@@ -4,15 +4,21 @@
 #include <time.h>
 #include "../include/justGarble.h"
 
-#define N_allels 10000
-//N_AB
+//two main parameters of LD test
+#define M_allels 10000
+
+//N_AB, N_Ab, N_aB, N_ab
 #define Size_I 8
```

Comment:
Can you add a comment explaining what `Size_I` is?

Reply:
I add a comment about Size_I